src/selectors/expandThoughts.ts

```diff
@@ -80,6 +80,10 @@ function expandThoughtsRecursive(state: State, expansionBasePath: Path, path: Pa
   const simplePath = !path || path.length === 0 ? HOME_PATH : simplifyPath(state, path)
   const thoughtId = head(path)
   const thought = getThoughtById(state, thoughtId)
+  if (!thought) {
+    console.error(new Error('expandThoughtsRecursive: Invalid Thought; No thought found with id ' + thoughtId))
+    return {}
+  }
```

Comment:
Since `thought` is only used in one place, let's only short circuit in that case rather than short circuiting the whole function:

```ts
  const childrenUnfiltered = showContexts
    ? thought ? childIdsToThoughts(state, getContexts(state, thought.value)) : []
    : ...
```